tests/test_data/shelikesfood.com/shelikesfood_2.json

```diff
@@ -0,0 +1,106 @@
+  "ingredient_groups": [
+    {
+      "ingredients": [
+        "7-8 cups diced rainbow colored vegetables of choice OR",
+        "1 large red bell pepper, diced",
+        "1 1/2 cups diced butternut squash, I like to use frozen",
+        "1/2 cup diced carrot",
+        "1 large yellow bell pepper, diced",
+        "1 cup small broccoli florets",
+        "1 cup quartered brussels sprouts",
+        "1 cup diced rutabaga",
+        "1/2 small red onion, diced",
+        "3 tablespoons olive oil, or oil of choice",
+        "1 1/2 teaspoons salt",
+        "1 teaspoon garlic powder",
+        "1/2 teaspoon black pepper",
+        "2 cups chickpeas",
+        "Quinoa for serving, or other grain of choice",
+        "Fresh herbs for garnish"
+      ],
+      "purpose": null
+    },
+    {
+      "ingredients": [
+        "1/2 cup tahini",
+        "3 tablespoons fresh lemon juice",
+        "4 tablespoons fresh orange juice",
+        "1/2 teaspoon lemon zest",
+        "1/2 teaspoon orange zest",
+        "1-2 clove garlic, minced",
+        "1 teaspoon pure maple syrup",
+        "3–6 tablespoons water, depending on how thick you would like the dressing",
+        "1/2 teaspoon salt, or more if needed"
+      ],
+      "purpose": "Citrus Tahini Sauce"
+    }
+  ],
+  "instructions_list": [
+    "Pre-heat oven to 425 degrees F. Add all of your diced vegetables to an extra large sheet pan and drizzle them with olive oil. Add garlic powder, salt and pepper (and any other seasonings you may want) and use your hands to toss everything together until vegetables are fully coated and arrange them in a single layer on the baking sheet.",
+    "Place veggies in the oven and roast until fork tender and beginning to caramelize, about 30-35 minutes, flipping once. Remove from the oven and let cool for a couple of minutes.",
+    "Make the tahini sauce while the vegetables are roasting. Add all sauce ingredients, except for the water, to a medium sized bowl, or jar, and mix until combined. Once the sauce gets thick, add a couple tablespoons of water at a time and mix until desired thickness is reached. You could also add in extra citrus juice instead of water, if you like.",
+    "Assemble you’re bowls. To each one, add about 3/4 cup quinoa, 1/2 cup chickpeas and 1/4 of the roasted veggies. Drizzle with desired amount of dressing and add extra salt or fresh herbs, if desired."
+  ],
+  "category": "Dinner",
+  "yields": "4 servings",
+  "description": "If you're looking to add more vegetables into your diet this year, these easy Roasted Rainbow Veggie Bowls with Citrus Tahini Sauce are the perfect way! These healthy, plant based bowls are customizable to your taste and great served with, protein packed, quinoa and chickpeas. The citrus tahini dressing brightens everything up adds creaminess.",
+  "total_time": 55,
+  "cook_time": 35,
+  "prep_time": 20,
+  "cuisine": "American",
+  "cooking_method": "Oven",
+  "nutrients": {
+    "servingSize": "1/4",
```

Comment:
The serving size of `1/4` here looks unusual; this does appear in the `schema.org` metadata on the origin recipe webpage though - so I don't know if there is much we can do about that.